components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -605,40 +603,34 @@ def generic_start_scheduler(
     ]
     # fmt: on
 
-    necessary_env_vars = [
-        f"PYTHONPATH={clp_site_packages_dir}",
-        (
-            f"BROKER_URL=amqp://"
-            f"{container_clp_config.queue.username}:{container_clp_config.queue.password}@"
-            f"{container_clp_config.queue.host}:{container_clp_config.queue.port}"
-        ),
-        (
-            f"RESULT_BACKEND=redis://default:{container_clp_config.redis.password}@"
-            f"{container_clp_config.redis.host}:{container_clp_config.redis.port}/"
-            f"{container_clp_config.redis.query_backend_database}"
+    env_vars = [
+        *generate_common_environment_variables(),
+        *generate_credential_environment_variables(
+            container_clp_config, include_db_credentials=True
         ),
+        *generate_celery_connection_environment_variables(container_clp_config),
         f"CLP_LOGS_DIR={container_logs_dir}",
         f"CLP_LOGGING_LEVEL={clp_config.query_scheduler.logging_level}",
```

Comment:
seems we have a misconfiguration in the original code where a compression scheduler's log level is set with the configured query scheduler's log level.

@coderabbitai create an issue to track this